emuu-client/src/components/UserProfile/Feeds.js

```diff
@@ -14,16 +14,16 @@ import {
   query,
   where,
 } from "firebase/firestore";
-
-function Feeds() {
+import { Link } from "react-router-dom";
+function Feeds({ setVideo }, { setUserProfile }) {
   const [recentVideos, setRecentVideos] = useState([]);
   const displayName = localStorage.getItem("displayName");
 
   async function getVideos() {
     //Get all video data
-    const docRef = collection(db, "Users", displayName, "Videos");
-    const querySnapshot = await getDocs(docRef);
-
+    const docRef = collection(db, "Videos");
```

Comment:
Good job of getting rid of the sub collection for users. It was unnecessary to store the same data twice and update them twice.